src/Traits/Connector/SendsRequests.php

```diff
@@ -74,15 +74,7 @@ public function send(Request $request, MockClient $mockClient = null, callable $
 
                 $response = $pendingRequest->executeResponsePipeline($response);
 
-                // We'll check if our tries is greater than one. If it is, then we will
-                // force an exception to be thrown if the request was unsuccessful.
-                // This will then force our catch handler to retry the request.
-
-                if ($maxTries > 1) {
-                    $response->throw();
-                }
-
-                return $response;
+                return $response->throw();
```

Comment:
I think this logic needs to remain the same because by default, Saloon does not throw exceptions if the request has failed. This check was in place because if no one has configured the send and retry, than the "max tries" is only 1. If this is the case, we shouldn't throw an exception here.

Reply:
By default it does not throw an exception, but if you want to throw an exception (using `shouldThrowRequestException` method), then the current code will not do that.

Reply:
@Sammyjo20 could you check my response here? ⬆️